scripts/setup_idf_macos.sh

```
@@ -52,6 +66,10 @@ ensure_brew_pkg libusb
 ensure_brew_pkg libffi
 ensure_brew_pkg openssl@3
 
+export ESP_PYTHON="$(pick_python)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)